app/views/admin/_metadata_summary_card.html.erb

```diff
@@ -18,36 +18,40 @@
   rows: [
     ({
       key: "Title of the finder",
-      value: schema["name"]
-    } if !(defined? previous_schema) || schema["name"] != previous_schema["name"]),
+      value: schema.name
+    } if !(defined? previous_schema) || schema.name != previous_schema.name),
```

Comment:
Going to see if I can figure out a better way of doing this using "[dirty](https://api.rubyonrails.org/classes/ActiveModel/Dirty.html)" attributes